FWCore/PythonParameterSet/src/PyBind11Wrapper.cc

```diff
@@ -1,9 +1,29 @@
 #include "FWCore/PythonParameterSet/interface/PyBind11Wrapper.h"
 #include "FWCore/Utilities/interface/Exception.h"
-//#include <iostream>
+#include "FWCore/Utilities/interface/EDMException.h"
+#include <iostream>
 namespace edm {
 
   void pythonToCppException(const std::string& iType, const std::string& error) {
+    auto colon = error.find(':');
+    if (colon != std::string::npos) {
+      if (error.substr(0, colon) == "EDMException") {
+        auto errorNameStart = error.find('{');
+        auto errorNameEnd = error.find('}');
+        if (errorNameStart != std::string::npos and errorNameEnd != std::string::npos) {
+          auto errorName = error.substr(errorNameStart + 1, errorNameEnd - errorNameStart - 1);
+          std::cout << "EDMException '" << errorName << "'" << std::endl;
+          if ("Configuration" == errorName) {
```

Comment:
Would it be feasible to generalize the name-to-error code translation by exploiting the information in
https://github.com/cms-sw/cmssw/blob/52c0dacc1af916e537cb419a39ec5845ae5087bf/FWCore/Utilities/src/EDMException.cc#L9-L10
?

On the other hand, the number of error codes to throw from the python code is likely small (I can think of one to add, but not really more)